### PR TITLE
Added majority of the markup including all the sections: portfolio-ca…

### PR DESCRIPTION
**Pull request by tkz96**
…rds, about-myself, languages, contact-form. Fixed some CSS errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,6 +51,155 @@ <h1>I am Talha.<br>Glad to see you!</h1>
...
+                </div>
+            </div> 
+            
+        </section>
+
+        <section class="about-myself">
+            <h2>About Myself</h2>
+            <p>Hello. I’m a software developer! I can help you build a product, feature or website, look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="lets-connect">LET'S CONNECT</p>
+            <div class="icons">
```
Comment (RolandM99) [inline index.html]: - [ ] the sentence **LET'S CONNECT** should use the header `hx` tag(h1, h2, h3,...). Kindly replace the `p` tag by a header ` hx` tag .

File: index.html [hunk 2]
```diff
@@ -51,6 +51,155 @@ <h1>I am Talha.<br>Glad to see you!</h1>
...
+                        <a href="https://www.medium.com/"><img src="icons/icon-medium.svg" alt="Medium">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="card action"><button class="see-project">Get my resume</button></div>
+        <ul class="myself-list">
+            <li class="languages">Languages</li>
+            <li class="frameworks">Frameworks</li>
+            <li class="skills">Skills</li>
```
Comment (RolandM99) [inline index.html]: - [ ] Kindly, Make sure that your design looks similar to the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
The technology categories should look like this : 
FIGMA DESIGN 👍 
image
YOUR DESIGN 👎 

image
- [ ] Please, kindly fix this to meet the Figma design provided

File: styles.css [hunk 3]
```diff
@@ -62,12 +161,30 @@ p {
...
+  padding: 0;
+}
+
+.card h2 {
+  margin-bottom: 0;
+}
+
+.see-project {
+  color: #396df2;
+  border-radius: 8px;
```
Comment (RolandM99) [inline styles.css]: - [ ] Please, Add the button styling (Hover, enable, Active, Pressed ) for all your buttons implemented referencing to the Figma template.
Figma buttons styling :
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hello @tkz96 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @tkz96, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] It will be better to regroup all assets (icons, background images) in a folder you can call it `assets`, which will make your code cleaner.

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tkz96** [comment]
Thank you so much for the review @RolandM99 @Hamzaoutdoors 

I will try my best to improve and use best pratices

**Pull request merged**